data/ca/municipalities/Todd-Gloria-56537ee1-ee89-4b98-92f5-c9b314a1a83a.yml

```diff
@@ -0,0 +1,17 @@
+id: ocd-person/56537ee1-ee89-4b98-92f5-c9b314a1a83a
+name: Todd Gloria
+given_name: Kevin
+family_name: Faulconer
```

Comment:
I think I fixed this the last time around, but maybe you could take a second look? It seems like the given/family name just need to be changed, but maybe the email and contact info too.

Reply:
@csnardi 

I checked and looks like the correct/incoming mayor is Todd Gloria. 

Fixed now here:  

https://github.com/openstates/people/blob/24e92efde165b1ce6d1563bdba03837d6da711e6/data/ca/municipalities/Todd-Gloria-ddd6f9ac-ab35-44c7-8291-a5360bbcbb44.yml

Will let you resolve if looks good.

Reply:
190dc0d